msfdb

```diff
@@ -50,10 +58,13 @@ require 'msfenv'
 @options = {
     component: :all,
     debug: false,
+    msf_db_admin_pass: '',
+    msf_db_admin_username: 'postgres',
```

Comment:
Are these dead now with the connection string functionality?

Reply:
sure are, removing now